user/views.py

```diff
@@ -1,3 +1,258 @@
+            get_user_id     = User.objects.get(id=user_id).id
+            get_company_id  = Company.objects.get(id=company_id).id
+            like_filter     = Like.objects.filter (user_id = get_user_id , company_id=get_company_id)
+            
+            if get_user_id and get_company_id and not like_filter:
+                Like.objects.create(user_id = get_user_id , company_id=get_company_id)
+                return JsonResponse({"message":"SUCCESS"},status=201)
+            
+            return JsonResponse({"message":"id or like_duplication INVALID"},status=400)
+        
+        except Company.DoesNotExist:
+            return JsonResponse({"message":"user_id or company_id INVALID"},status=400)
+
+        except KeyError :
+            return JsonResponse({"message":"Key_Error"},status=400)
+
+    @token_check
+    def delete(self , request):
+        try:
+            data            = json.loads(request.body)
+            company_id      = data["company_id"]
+            user_id         = request.user.id
+
+            get_user_id     = User.objects.get(id=user_id).id
+            get_company_id  = Company.objects.get(id=company_id).id
+            
+            if get_user_id and get_company_id :
+                Like.objects.get(user_id = get_user_id , company_id=get_company_id).delete()
+                return JsonResponse({"message":"SUCCESS"},status=201)
+            
+            return JsonResponse({"message":"user_id or company_id INVALID"},status=400)
+        
+        except Like.DoesNotExist:
+            return JsonResponse({"message":"user_id or company_id INVALID"},status=400)
+
+        except KeyError :
+            return JsonResponse({"message":"Key_Error"},status=400)
+             
+
+    @token_check
+    def get(self , request):
+        user_id = request.user.id 
+        
+        company_list=[
+                { 
+                "id"            :applied_status.id , 
+                "title"         :applied_status.title ,
+                "name"          :applied_status.name,
+                "city"          :applied_status.district.city.name,   
+                "compensation"  :applied_status.compensation.recommender+applied_status.compensation.applicant,
+                "like_count"    :applied_status.likes_count,
+                "response_rate" :applied_status.response_rate.rate,
+                "image_url"     :applied_status.image_url   
+            } for applied_status in  User.objects.get(id=user_id).likes.select_related('district__city', 'compensation', 'response_rate')]
+
+        return JsonResponse( {"message":"SUCCESS","job_list":company_list},status=201)
+
+class TagView(View):
+   
+    @token_check 
+    def post(self , request):
+        data           = json.loads(request.body) 
+        user_id             = request.user.id   
+        get_user_id         = User.objects.get(id=user_id).id
+        filter_delete       = User.objects.get(id=user_id)
```

Comment:
같은 동작을 하는 코드가.... 있어보입니다.. 다시 한번 생각해보시고 수정해보세요!

```suggestion
        user_id                = request.user.id   
        get_user_id         = User.objects.get(id=user_id).id
        filter_delete         = User.objects.get(id=user_id)
```